storage/mysql/tree_storage.go

```diff
@@ -151,7 +151,7 @@ func decodeSignedTimestamp(signedEntryTimestampBytes []byte) (trillian.SignedEnt
 	return signedEntryTimestamp, nil
 }
 
-func encodeSignedTimestamp(signedEntryTimestamp trillian.SignedEntryTimestamp) ([]byte, error) {
+func EncodeSignedTimestamp(signedEntryTimestamp trillian.SignedEntryTimestamp) ([]byte, error) {
```

Comment:
Yes because implementations have to create them when leaves are queued. We could break the encode / decode out of here at some point it we liked.